packages/admin/src/appHeader/dropdown/AppHeaderDropdown.tsx

```diff
@@ -0,0 +1,70 @@
+import { ChevronDown, ChevronUp } from "@comet/admin-icons";
+import { Popover, useTheme } from "@material-ui/core";
+import { PopoverProps } from "@material-ui/core/Popover/Popover";
+import { StyledComponentProps } from "@material-ui/core/styles";
+import * as React from "react";
+
+import { mergeClasses } from "../../helpers/mergeClasses";
+import { AppHeaderAction, AppHeaderActionProps } from "../action/AppHeaderAction";
+import { CometAdminAppHeaderDropdownClassKeys, useStyles } from "./AppHeaderDropdown.styles";
+
+interface Props extends AppHeaderActionProps {
+    children?: ((closeDropdown: () => void) => React.ReactNode) | React.ReactNode;
+    actionContent?: React.ReactNode;
+    dropdownArrow?: ((isOpen: boolean) => React.ReactNode) | null;
+    popoverProps?: Partial<PopoverProps>;
+}
+
+function DefaultArrowUp(): React.ReactElement {
+    const { palette } = useTheme();
+    return <ChevronUp htmlColor={palette.primary.contrastText} />;
+}
+
+function DefaultArrowDown(): React.ReactElement {
+    const { palette } = useTheme();
+    return <ChevronDown htmlColor={palette.primary.contrastText} />;
+}
+
+export function AppHeaderDropdown({
+    children,
+    actionContent,
```

Comment:
I am not sure if the term "content" is ideal. 
What about `actionChildren`, or `buttonChildren`? ...not really perfect either.